docs/secure/user-auth.md

```diff
@@ -1,43 +1,36 @@
 # User authentication
 
 Percona Everest provides user authentication to ensure that only authorized users can access the databases hosted on Percona Everest. Securing access to these databases is crucial to safeguard sensitive data and restrict database access to authorized users only. 
-
-To access Percona Everest:
 {.power-number}
 
-1. Copy the authorization token that is displayed on the terminal while [installing Percona Everest](../install/installEverest.md).
+1. When you [install Percona Everest](../install/installEverest.md), an `admin` user is automatically created. To retrieve the password for the `admin` user, run the following command:
 
-    ![!image](../images/everest_user-auth-token.png)
+    ```sh
+    everestctl accounts initial-admin-password
+    ```
 
-2. Navigate to the Everest UI.
+    We strongly recommend that you reset the  initial `admin` password after installation.
 
-3. Enter the authorization token that you copied in step 1.
+    
+    !!! note " Note"
+        You can create a [new user account](manage_users.md#create-a-new-user) after installing Percona Everest. Once the account is created, you can then log in using your newly created user credentials.
 
-    ![!image](../images/everest_user-auth-login.png)
 
+2. Navigate to the Everest UI.
+
+3. Enter the password you received in step 1. If you changed the initial password, enter the updated one.
 
 4. Click **Log in**.
 
 
-## Reset authorization token
+## Reset initial password
```

Comment:
I don't think it's correct to call this resetting the initial password, this is setting a password for the admin user which will replace the automatically generated one.